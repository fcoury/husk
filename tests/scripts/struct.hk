struct Person {
    name: string,
    age: int,
}

let person = Person { name, age };
println(person.name);
println(person.age);
println(person);

person.age = 22;
println(person.age);
