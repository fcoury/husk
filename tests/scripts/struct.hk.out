Felipe
44
struct Person {name: String("Felipe"), age: Int(44)}
22
